index.html

```diff
@@ -15,38 +15,224 @@
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="art-name">Back End Dev</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="year">2015</p>
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card">
+        <img class="card-img" src="./images/Tonic2.png" alt="Tonic Project 2">
+        <div class="text-cont">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="title-elements">
+            <li>
+              <p class="pr-name">CANOPY</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="art-name">Back End Dev</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="year">2015</p>
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card">
+        <img class="card-img" src="./images/mps2.png" alt="Multi-Post Project 2">
+        <div class="text-cont">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="title-elements">
+            <li>
+              <p class="pr-name">CANOPY</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="art-name">Back End Dev</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="year">2015</p>
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card2">
```

Comment:
- [ ] For better code organization and HTML best practices, please consider using a `section` tag instead of `article` tag to improve user interaction and for better semantic HTML.